### PR TITLE
Update Dockerfiles for 1.19.0 release

### PR DESCRIPTION
**Pull request by lincoln-lil**
* Update Dockerfiles for 1.19.0 release

### CODE REVIEW

File: 1.19/scala_2.12-java11-ubuntu/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    rm -rf "$GNUPGHOME" flink.tgz.asc; \
+  fi; \
+  \
+  tar -xf flink.tgz --strip-components=1; \
+  rm flink.tgz; \
+  \
+  chown -R flink:flink .; \
+  \
+  # Replace default REST/RPC endpoint bind address to use the container's network interface \
+  CONF_FILE="$FLINK_HOME/conf/flink-conf.yaml"; \
```
Comment (MartijnVisser) [inline 1.19/scala_2.12-java11-ubuntu/Dockerfile:84]: Is this still correct, given the new YAML parser?

**Review by MartijnVisser — APPROVED** [review]
I think it's good, left one comment to double check

File: 1.19/scala_2.12-java11-ubuntu/Dockerfile (hunk 1 above)
Comment (lincoln-lil) [reply 1.19/scala_2.12-java11-ubuntu/Dockerfile:84]: Good catch! I checked both the FLINK-33297 and [FLIP-366](https://cwiki.apache.org/confluence/display/FLINK/FLIP-366%3A+Support+standard+YAML+for+FLINK+configuration),  the default configuration file has been changed to `config.yaml`. Also verified the dist package, we should use the new `config.yaml` instead.

Comment (lincoln-lil) [reply 1.19/scala_2.12-java11-ubuntu/Dockerfile:84]: I checked both the FLINK-33297 and FLIP-366, the default configuration file has been changed to `config.yaml`, but the script is compatible with the old `flink-conf.yaml` file. 
This is FLIP-366's expected behavior (For compatibility reasons, in Flink 1.x, if the old configuration file `flink-conf.yaml` exists in the Flink conf directory, Flink will ignore the new configuration file and use the old parser to parse `flink-conf.yaml` as the Flink configuration. In Flink 2.x, Flink will no longer support parsing the old configuration file `flink-conf.yaml`).

**Review by lincoln-lil — COMMENTED** [review]
Thanks @MartijnVisser for reviewing this!

**Pull request merged**